std_msgs/Header header

audio_utils/AudioFrame audio_frame

string[] audio_classes
float32[] audio_descriptor
float32[] voice_descriptor

float64 direction_x
float64 direction_y
float64 direction_z
